python/cudf/cudf/tests/test_concat.py

```diff
@@ -1793,3 +1793,48 @@ def test_concat_categorical_ordering():
     got = gd.concat([gdf, gdf, gdf])
 
     assert_eq(expect, got)
+
+
+@pytest.fixture(params=["rangeindex", "index"])
+def concat_index(request):
+    if request.param == "rangeindex":
+        return pd.RangeIndex(0, 4)
+    else:
+        return pd.Index(["a", "h", "g", "f"])
+
+
+@pytest.fixture(params=["dataframe", "series"])
+def concat_obj(request, concat_index):
+    if request == "dataframe":
+        return pd.DataFrame(
+            {
+                "b": [1, 2, 3, 4],
+                "d": [7, 8, 9, 10],
+                "a": [4, 5, 6, 7],
+                "c": [10, 11, 12, 13],
+            },
+            index=concat_index,
+        )
+    else:
+        return pd.Series([4, 5, 5, 6], index=concat_index)
```

Comment:
This seems like a reasonable use of fixtures to me, except `concat_index` and `concat_obj` seem like very general names for fixtures that are only ever used by a single test. If the intent is to use them more generally, should they appear near the top of this file?

Reply:
I'm not sure they're more generic (renamed to `singleton_XXX`)